modules/dxp/apps/portal-search-tuning/portal-search-tuning-rankings-web/src/main/java/com/liferay/portal/search/tuning/rankings/web/internal/upgrade/v1_0_0/RankingUpgradeProcess.java

```diff
@@ -0,0 +1,76 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.portal.search.tuning.rankings.web.internal.upgrade.v1_0_0;
+
+import com.liferay.petra.string.StringBundler;
+import com.liferay.petra.string.StringPool;
+import com.liferay.portal.kernel.dao.jdbc.AutoBatchPreparedStatementUtil;
+import com.liferay.portal.kernel.service.ClassNameLocalService;
+import com.liferay.portal.kernel.upgrade.UpgradeProcess;
+import com.liferay.portal.search.tuning.rankings.web.internal.constants.ResultRankingsConstants;
+import com.liferay.portal.search.tuning.rankings.web.internal.index.Ranking;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+
+/**
+ * @author Almir Ferreira
+ */
+public class RankingUpgradeProcess extends UpgradeProcess {
+
+	public RankingUpgradeProcess(ClassNameLocalService classNameLocalService) {
+		_classNameLocalService = classNameLocalService;
+	}
+
+	@Override
+	protected void doUpgrade() throws Exception {
+		try (PreparedStatement preparedStatement1 = connection.prepareStatement(
+				"select jsonStorageEntryId, valueString from " +
+					"jsonStorageEntry where classNameId = ? and key_ = ?");
+			PreparedStatement preparedStatement2 =
+				AutoBatchPreparedStatementUtil.concurrentAutoBatch(
+					connection,
+					"update jsonStorageEntry set key_ = ?, valueString = ? " +
+						"where jsonStorageEntryId = ?")) {
+
+			preparedStatement1.setLong(
+				1, _classNameLocalService.getClassNameId(Ranking.class));
```

Comment:
im kind of thinking we should do the upgrade that changes the package of Ranking in https://liferay.atlassian.net/browse/LPS-204773 before we do this upgrade... feels like there's potential for the name to be out of sync if we did this one first. and actually maybe should be hardcoding the FQCN here as a string instead of passing Ranking.class